pkg/runtime-tools/generate/generate.go

```diff
@@ -332,6 +335,16 @@ func (g *Generator) AdjustOomScoreAdj(score *nri.OptionalInt) {
 	}
 }
 
+func (g *Generator) AdjustNamespaces(namespaces []*nri.LinuxNamespace) error {
+	for _, ns := range namespaces {
+		if err := g.AddOrReplaceLinuxNamespace(ns.Type, ns.Path); err != nil {
+			return err
+		}
+	}
```

Comment:
In the commit you say `We are interested in running some parts of a pod in host` but if I understand this correctly you can't remove a namespace, correct ?